pkgs/applications/graphics/ImageMagick/default.nix

```diff
@@ -13,7 +13,7 @@
 , libjpegSupport ? true, libjpeg
 , djvulibreSupport ? true, djvulibre
 , lcms2Support ? true, lcms2
-, openexrSupport ? !stdenv.hostPlatform.isMinGW, openexr
+, openexrSupport ? !stdenv.hostPlatform.isMinGW, openexr_3
```

Comment:
FYI, https://github.com/NixOS/nixpkgs/pull/375773#discussion_r1925252383 : we shouldn't change the override interface and instead change the version in `all-packages.nix` with `override`. It will only make #367406 diff smaller in the end!

Reply:
Hum OK but it won't make the diff smaller (overrides will need to be removed) and it will generate extra work because we will have to add back the definition in `all-packages.nix` of some of the packages under `by-name`. Not sure that's the right call for this situation.

Reply:
Might not be. In my case, there was already an entry in `all-packages.nix`. Your mileage may vary.